code/modules/assembly/flash.dm

```diff
@@ -293,9 +293,14 @@
 		var/mob/living/silicon/robot/R = M
 		log_combat(user, R, "flashed", src)
 		update_icon(1)
-		R.Paralyze(70)
-		R.flash_act(affect_silicon = 1, type = /atom/movable/screen/fullscreen/flash/static)
-		user.visible_message("<span class='disarm'>[user] overloads [R]'s sensors with the flash!</span>", "<span class='danger'>You overload [R]'s sensors with the flash!</span>")
+		if(R.last_flashed + 30 SECONDS < world.time)
+			R.last_flashed = world.time
+			R.Paralyze(5 SECONDS)
+			R.flash_act(affect_silicon = 1, type = /atom/movable/screen/fullscreen/flash/static)
+			user.visible_message("<span class='disarm'>[user] overloads [R]'s sensors with the flash!</span>", "<span class='danger'>You overload [R]'s sensors with the flash!</span>")
+		else
+			user.visible_message("<span class='disarm'>[user] attempts to overload [R]'s sensors with the flash, but it has no effect!</span>", "<span class='danger'>You attempt to overload [R]'s sensors with the flash, but they have already been overloaded recently!</span>")
```

Comment:
This should at least have some effect like blinding or slowing them

Reply:
I really think this should be either or:

Either reduce the immunity period OR make it so they're still blinded but not stunned during the period.

Reply:
Still blinded but not stunned seems okay, since thats the effect of the flash on humans

Reply:
Flashes now do the full blinding effect even when they don't stun 👍